packages/api/src/microsoft/teams/api/clients/conversation/activity.py

```diff
@@ -39,16 +39,13 @@ async def create(self, conversation_id: str, activity: ActivityParams) -> SentAc
         Returns:
             The created activity
         """
+
         response = await self.http.post(
             f"{self.service_url}/v3/conversations/{conversation_id}/activities",
             json=activity.model_dump(by_alias=True, exclude_none=True),
         )
 
-        if activity.type == "typing":
-            # NOTE: Service returns empty response
-            return SentActivity(id="DO_NOT_USE_PLACEHOLDER_TYPING_ID", activity_params=activity)
-
-        id = response.json()["id"]
+        id = response.json().get("id", "DO_NOT_USE_PLACEHOLDER_TYPING_ID")
```

Comment:
could you keep a note that the service returns an empty response for a non-streaming typing activity?